### DISCUSSION TITLE
Failed XMR - BTC , Atomic swap, Coins lost, stolen ?

### DISCUSSION THREAD

**Discussion by scott-green007** [comit-network/xmr-btc-swap#868] [Q&A] [answered]
Unstoppableswap which claims to be trustless has taken my coins. After sending 0.01 BTC, I was then told the other wallet does not have enough XMR to complete the transaction, my coins were never returned. At present they are stuck in this wallet. The sender address is also my refund address, if someone can do this for me manfully please help and don't steal my coins, Here is a link to the blockchain: https://www.blockchain.com/btc/address/bc1qzan405hsggd2egdmrevd9ez00q53drfhnj4n5s

**Comment by binarybaron** [comment, marked as answer]
No one has stolen your coins. Run `./swap balance` and `./swap withdraw-btc --address YOUR-ADDRESS` to withdraw your funds from the internal wallet.

**Comment by scott-green007** [comment]
Hello,

I'm using this https://unstoppableswap.net/ I can only repeat the same process. I put in my BTC refund address and my XMR address next copy this command seen below and place it into powershell. After I sent funds I was told the other wallet didn't have enough XMR please try again later and my 0.01 BTC has been stuck in this third party address for around three days: https://www.blockchain.com/btc/address/bc1qzan405hsggd2egdmrevd9ez00q53drfhnj4n5s -- When I repeat this process it gives this info below in powershell, You said swap balance and ./swap withdraw-btc -- There is no swap withdraw button, I can't see it anywhere unless its on a different program or am I missing something? I'm doing this on the same PC and I keep getting the same info in powershell. I tried pasting this command in another PC it won't activate any transfer. If I try the same transfer using the same BTC/XMR addresses on a different PC its asking to send to a different BTC address. Thank you.

Starting swap. This can take some time...
Connected to Alice at /dnsaddr/battery.commitcri.me/p2p/12D3KooWCSJUBJ3DkxAdwE1rX5ejBWttexYWsgqGBTStX47rU71G
Received quote price=0.00398105 BTC minimum_amount=0.00010000 BTC maximum_amount=0.01000000 BTC
Starting new swap amount=0.00999784 BTC fees=0.00000216 BTC swap_id=23e88a1c-e963-4583-a553-21cfa88cb86f
Estimated fee of 241.3940 is smaller than the min relay fee, defaulting to min relay fee 1000
Estimated fee of 262.5380 is smaller than the min relay fee, defaulting to min relay fee 1000
Error: Failed to complete swap

Caused by:
Seller's XMR balance is currently too low to fulfill the swap request to buy 0.00999784 BTC, please try again later

PS C:\Users\User\swaptool>

[image.png]

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by binarybaron** [comment]
Assuming you are on windows:

1. Open PowerShell
2. Type ` cd ~/swaptool` and hit enter
3. Type `./swap balance` and hit enter. It will display the balance of the internal bitcoin wallet
4. Finally, run `./swap withdraw-btc --address YOUR_ADDRESS` to withdraw your BTC to your own wallet

**Comment by scott-green007** [comment]
Hello,

Thanks for trying to help, I will tell you what I did. I copied and pasted this into powershell below,

~/swaptool

I pressed enter

I then copied and pasted this,

./swap balance

I pressed enter.

I went to https://unstoppableswap.net/ using the same PC put in the same BTC and XMR addresses, I hit the command button copied and pasted that info into powerhsell along with the info you gave me its not working ?

What does this mean below ? Is this what I'm doing on unstoppable ? There is no withdraw button. I have being trying so hard with what you said it is not working.

4. Finally, run ./swap withdraw-btc --address YOUR_ADDRESS to withdraw your BTC to your own wallet

Thank you.

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐

**Comment by scott-green007** [comment]
I was able to complete the transaction. I kept on trying, Alice detected XMR this time and my BTC in the third party wallet was exchanged for XMR.

Alice said try again later after the first failure. I thought it would never work again now I'm very happy. I think the system needs to detect enough XMR before a BTC address is given to send funds or even better the atomic swap happens with my own BTC address in a wallet backed by a seed.

Sent with [ProtonMail](https://protonmail.com/) Secure Email.

‐‐‐‐‐‐‐ Original Message ‐‐‐‐‐‐‐
